src/OSmOSE/utils/formatting_utils.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+
+
+def aplose2raven(df: pd.DataFrame) -> pd.DataFrame:
+    """Export an APLOSE formatted result file to Raven formatted DataFrame
+
+    Parameters
+    ----------
+    df: APLOSE formatted result DataFrame
+
+    Returns
+    -------
+    df2raven: Raven formatted DataFrame
+
+    Example of use
+    --------------
+    aplose_file = Path("path/to/aplose/result/file")
+    df = (
+        pd.read_csv(aplose_file, parse_dates=["start_datetime", "end_datetime"])
+        .sort_values("start_datetime")
+        .reset_index(drop=True)
+    )
+
+    df_raven = aplose2raven(df)
+
+    # export to json format
+    df.to_json("output/path.json")
+
+    # import json format
+    df_from_json = pd.read_json("path/to/json/file")
```

Comment:
Is it related to `aplose2raven`? It is not clear whether `df_from_json` is supposed to be a raven-formatted dataframe or an APLOSE-formatted dataframe.